index.html

```diff
@@ -195,6 +195,32 @@ <h6 class="about-title">
       </div>
     </div>
   </section>
+  <footer>
+    <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+    <form action="">
```

Comment:
### Missing Requirements :memo:
![missing-req-contact-form](https://user-images.githubusercontent.com/93039359/192786008-8b36838e-680e-4269-b586-f2741af89f6e.png)


- [ ] You didn't implement formspree in your contact form and that's a requirement in your project. Kindly implement the [Formspree service](https://formspree.io/html) to make your contact form work. :+1: